languages/ZunitConfig.groovy

```diff
@@ -95,7 +95,12 @@ buildUtils.createLanguageDatasets(langQualifier)
 	zUnitRunJCL.saveOutput(logFile, props.logEncoding)
 
 	// Store Report in Workspace
-	new CopyToHFS().dataset(props.zunit_bzureportPDS).member(member).file(reportLogFile).hfsEncoding(props.logEncoding).append(false).copy()
+	try {
```

Comment:
I think, we should move this into the section where the jobRC is evaluated. Only with a valid RC, we should try to store it. Otherwise we might retrieve a previously generated report.